models/help_request.js

```diff
@@ -2,23 +2,33 @@ var database = require("../database/database");
 var mongoose = database.getmongoose();
 var Schema = mongoose.Schema;
 
+const arrayLimit = (val) => {
+  return val.length <= 6;
```

Comment:
Somehow it's letting me add 7 entries

Reply:
And this doesnt actually send an error as a response. It still sends back 200 but logs the error. Maybe we need to find a new way of handling this

Reply:
Does it add 7 entries in the database or does it just not throw an error in the http response?

Reply:
OK it seems to be just not throwing an error but doesnt actually add it

Reply:
Changed appropriately